### ISSUE TITLE
Using the correct neutrino mass loss in BH formation

### ISSUE THREAD

**Issue by michaelzevin** [COSMIC-PopSynth/COSMIC#357]
We need to figure out from Chris Fryer what the correct amount of neutrino mass loss to use.

**Comment by michaelzevin** [comment]
After talking with Chris Fryer and Vicky, we decided it would be best to make this a flag (@katiebreivik @scottcoughlin2014 let me know if you disagree). 

BHmassloss = -1: you lose 10% of CO core mass if CO core mass is less than 5 Msun, else lose 0.1*5 = 0.5 Msun
BHmassloss \exists [0,1]: you lose CO core mass * BHmassloss

What do you think? We can also change this variable name to whatever.

**Comment by michaelzevin** [comment]
This is addressed in PR #374. Will close this issue ones that PR is reviewed and merged.

**Comment by michaelzevin** [comment]
Fixed in #374
